packages/ui/src/components/va-alert/useAlertStyles.ts

```diff
@@ -20,6 +20,8 @@ type AlertStyleProps = {
 
 export const useAlertStyles = (props: AlertStyleProps) => {
   const dark = 'var(--va-dark)'
+  const white = '#ffffff'
+  const paddingDense = '0.25rem 0.75rem'
```

Comment:
And here we can add local css-variable in `_variables.scss`.
something like this: `--va-affix-padding-dense: '0.25rem 0.75rem';`

Reply:
done